feature-libs/order/styles/components/my-account-v2/_my-account-v2-order.scss

```diff
@@ -0,0 +1,106 @@
+%cx-my-account-v2-orders {
+  margin: 1rem 0rem;
+  border: 1px solid var(--cx-color-medium);
+  .cx-myaccount-view-header {
```

Comment:
[MINOR]
As we agreed in the [comment under PR 1](https://github.com/SAP/spartacus/pull/17976#discussion_r1360511058): for consistency reasons, let's keep "my account" as 2 separate in our codebase (especially in the new code which is part of our public API)

Reply:
done